src/sentry/static/sentry/app/views/organizationIncidents/details/incidents.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import SentryTypes from 'app/sentryTypes';
+import NavTabs from 'app/components/navTabs';
+import Link from 'app/components/link';
+
+import {PageContent} from 'app/styles/organization';
+import theme from 'app/utils/theme';
+
+import IncidentsSuspects from './suspects';
+import Activity from './activity';
+import RelatedIncidents from './relatedIncidents';
+
+const TABS = {
+  activity: {name: t('Activity'), component: Activity},
+  related: {name: t('Related incidents'), component: RelatedIncidents},
+};
+
+export default class Incidents extends React.Component {
+  static propTypes = {
+    incident: SentryTypes.Incident,
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeTab: Object.keys(TABS)[0],
+    };
+  }
+  handleToggle(tab) {
+    this.setState({activeTab: tab});
+  }
+
+  render() {
+    const {incident} = this.props;
+    const {activeTab} = this.state;
+    const ActiveComponent = TABS[activeTab].component;
+
+    return (
+      <StyledPageContent>
+        <Main>
+          <PageContent>
+            <NavTabs underlined={true}>
```

Comment:
You can leave off the `={true}`. `<navTabs underlined>` works

Reply:
Yeah, I wanted to be explicit about it's value